### PR TITLE
Adds a jumping ability to player-controlled roos, also more roos

### PR DESCRIPTION
**Pull request by totallynotkatfood**
## About The Pull Request

Ported jumping abilities from para with some mods for the roos

## Why It's Good For The Game

s p a c e  r o o s

## Changelog
:cl:
add: Player controlled roos can now jump
add: Added more kangaroo types
/:cl:

### CODE REVIEW

**Comment by ZombiesVsPlants** [comment]
SPACE ROOS

**Comment by coolsurf6** [comment]
Time to make a kangaroo race.

File: austation/code/datums/mutations/jumpy.dm [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		container.visible_message("<span class='danger'>[user.loc]</b> emits a loud thump and rattles a bit.</span>")
+		playsound(user.loc, 'sound/effects/bang.ogg', 50, 1)
+		var/wiggle = 6
+		while(wiggle > 0)
+			wiggle--
+			container.pixel_x = rand(-3,3)
+			container.pixel_y = rand(-3,3)
+			sleep(1)
+		container.pixel_x = 0
+		container.pixel_y = 0
```
Comment (MCterra10) [inline austation/code/datums/mutations/jumpy.dm:72]: learn2use animate() lmao

Comment (totallynotkatfood) [reply austation/code/datums/mutations/jumpy.dm:72]: not my code lmao

Comment (TheFakeElon) [reply austation/code/datums/mutations/jumpy.dm:72]: Should at least say where you ported it from if it’s not yours then smh

Comment (MCterra10) [reply austation/code/datums/mutations/jumpy.dm:72]: ^
please credit code ports somewhere in your PR.

**Pull request merged**